sci-electronics/splat/splat-1.4.2.ebuild

```diff
@@ -14,26 +14,28 @@ SLOT="0"
 KEYWORDS="~amd64 ~ppc ~x86"
 IUSE="doc hires l10n_es"
 
-DEPEND="sys-libs/zlib
-	app-arch/bzip2"
+DEPEND="
+	app-arch/bzip2
+	sys-libs/zlib
```

Comment:
only needed at compile time? seems odd...

Reply:
It's included in RDEPEND 2 lines below ;)